item-spec/json-schema/item.json

```diff
@@ -12,7 +12,10 @@
   ],
   "definitions": {
     "common_metadata": {
-      "anyOf": [
+      "allOf": [
```

Comment:
You still need to change/remove lines 126-131 in this file.

Reply:
got it, thanks!